_pages/how-we-work/guides/consulting-engineer-guide.md

```diff
@@ -0,0 +1,99 @@
+
+### Consulting support
+
+Everyone at 18F who works with partners can consider themselves “in consulting.” This means there’s a lot of value to be found in project or chapter discussions across the whole organization. Don’t hesitate to join Slack channels or attend meetings that will help you broaden your experience with 18F projects. The below resources will offer a lot of context for the partner-facing and organizational challenges of consulting projects:
+
+* [#path-analysis](https://app.slack.com/client/T025AQGAN/C3J7P5MMK)
+* [Project Artifacts repo](https://github.com/18F/path-analysis)
+* [#project-resources](https://app.slack.com/client/T025AQGAN/CHZLJBWCV)
+* [#wg-consulting](https://app.slack.com/client/T025AQGAN/CDVJ9U4K1) (and its regular meeting)
+
+### Technical support
+If it’s a technical issue outside your wheelhouse, talk with our specialists. They’ll be happy to help you find a solution:
+
+* Engineering Practices Guild: Ask in [#dev](https://app.slack.com/client/T025AQGAN/C02CD5VUQ)
+* DevOps Guild: Ask in [#g-devops](https://app.slack.com/client/T025AQGAN/C029KQMBZ)
+* Security & Compliance Guild: Ask in [#g-security-compliance](https://app.slack.com/client/T025AQGAN/CG8SNAEQM)
+
+Don’t hesitate to call on the Acquisitions, Content and Design Chapters as needed, either. They have guilds and labs specifically aimed at helping you.
+
+Also, don’t forget about our portfolio of existing solutions like Cloud.gov, Login.gov, and Federalist who may be able to offer useful services to your partner. If you think they might be able to help, talk to them.
+
+## Working with your team and partner
+First, keep in mind the principles outlined in How We Relate to Partners and always work with trust, respect, value, and collaboration. As the technical specialist on the team, also realize that people will look to you as the expert in the room and expect you to quickly understand the partner’s problems, land on potential solutions, and to be able to present them in a respectful and helpful manner. 
+
+The classic engineering belief that being super blunt is always the best approach is probably not going to be effective. A little diplomacy is very helpful. That said, trust your instincts and don’t be afraid to gently disagree with the partner.
+
+You’ll also need to respect the diversity of skills and backgrounds on the engagement, both on our side as well as the partner’s side. You may be the only software person the partner engages with, so it’s important to respect and value all the other perspectives you’ll encounter. The problem that a partner describes may not be the one they really need to solve, but it’s the one they experience, so honor that.
+
+Also remember that you’re not doing this alone!  You’ll be able to bounce ideas off others on your engagement team as well as ask for feedback from the engineering chapter (#dev in Slack).
```

Comment:
I'd suggest moving the Technical Support section to here. "working with partners" is arguably more relevant, and this last graf gives a natural lead-in.

Reply:
Ok. I moved the whole section with resources to keep that coherent and structured well.